.github/workflows/clang.yaml

```diff
@@ -159,7 +159,7 @@ jobs:
 
       - name: Merge Test Results
         run: |
-          junitparser merge --glob artifacts/**/twister.xml junit.xml
+          junitparser merge --glob "artifacts/*/twister.xml" "artifacts/*/*/twister.xml" junit.xml
```

Comment:
use single quote, otherwise bash is going to expand it, still works but not ideal